Vagrantfile

```diff
@@ -17,6 +17,8 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
   config.vm.box_version = "> 0.3.2.1714"
 
   config.vm.provision :shell, :inline => $bootstrap, :privileged => true
+  # Use the git configuration from the host in the VM
+  config.vm.provision "file", source: "~/.gitconfig", destination: ".gitconfig"
```

Comment:
This gives:

```
There are errors in the configuration of this machine. Please fix
the following errors and try again:

File provisioner:
* File upload source file /Users/Adam/.gitconfig must exist
```

What is ideal? Do you think that it is best to:
* Have the error as it stands
* If the config doesn't exist there, ask the user where it does exist
* Ignore the error and continue without copying any config file if one does not exist at `~/.gitconfig`
?